src/scss/_bases.scss

```diff
@@ -54,7 +60,7 @@ article {
 }
 
 body {
-    color: #73879C;
+    color: $default-light-blue;
     background: #2A3F54;
```

Comment:
@draecko can we create a variable file and place it on top ? like bootstrap do ? 
I think that's @atais thought ???